Podcast/ActionSheetViewController.swift

```diff
@@ -3,19 +3,51 @@ import UIKit
 
 class ActionSheetTableViewCell: UITableViewCell {
     
+    let padding: CGFloat = 18
+    var leftPadding: CGFloat = 50
+    var titleLabel: UILabel!
+    var iconImage: UIImageView!
+    
     override init(style: UITableViewCellStyle, reuseIdentifier: String?) {
```

Comment:
![image](https://user-images.githubusercontent.com/20246620/32423780-8f228d80-c276-11e7-902f-a60549c29498.png)
the tableviewcells have a slightly different background color than the rest of the actionsheet

Reply:
fixed